index.html

```diff
@@ -0,0 +1,168 @@
+            <li><a href="#">Events</a></li>
+            <li><a href="#">Launched</a></li>
+            <li class="last-item"><a href="#">Library</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <!-- Mobile Menu -->
+    <section class="mobile-menu">
+      <button class="x-mark">&times;</button>
+      <ul id="menu-list">
+        <li><a id="" href="index.html">Home</a></li>
+        <li><a id="" href="about.html">About</a></li>
+        <li><a id="" href="#">Contact</a></li>
+      </ul>
+    </section>
+    <!-- Home page Intorduction -->
+    <main class="bg-gray">
+      <section class="introduction home-intro bg-gray font">
+        <h1>"Unleash your curiosity at the library."</h1>
+        <h2>ABREHOT Library</h2>
+        <div class="p-2 border-1 bg-gray">
+          <p class="p-1">
+            "Embark on a journey of discovery and endless learning as you
+            immerse yourself in the boundless treasures of knowledge and
+            imagination available at your fingertips in our library."
+          </p>
+        </div>
+        <p class="tbm-header bg-gray">25<sup>th</sup> May 2023</p>
+        <p class="nyerere">@Arat Killo, Addis Ababa</p>
+      </section>
+      <!-- Home page Main Part -->
+      <section class="main-part font">
+        <h2>OUR SERVICES</h2>
+        <div class="dash">_____</div>
+        <div class="main-div">
+          <div>
+            <img src="images/book_borrowing.png" alt="book Logo" />
+            <h3>Borrow</h3>
+            <p>Books are a uniquely portable magic.Enjoy by being Member.</p>
+          </div>
+          <div>
+            <img src="images/audio-book.png" alt="Audiobook Logo" />
+            <h3>Audio Book</h3>
+            <p>
+              An audiobook is a great way to escape into a story while you're
+              doing other things.
+            </p>
+          </div>
+          <div>
+            <img src="images/place.jpg" alt="place" />
+            <h3>Corner</h3>
+            <p>
+              The best place to study, quiet, well-lit, comfortable, and free
+              from distractions.
+            </p>
+          </div>
+          <div>
+            <img class="bg-red" src="images/coffee.jpg" alt="Coffee" />
+            <h3>Coffee</h3>
+            <p>
+              When you drink coffee ideas march in like the army.Enjoy our
+              Coffee.
+            </p>
+          </div>
+          <div>
+            <img src="images/internet.jpg" alt="Internet" />
+            <h3>Internet</h3>
+            <p>
+              Enjoy our High Bandwidth Internet.we support wifi and Ethernet.
+            </p>
+          </div>
+        </div>
+        <div class="last-main-div">
+          <h3 class="bg-red h3-one">LIBRARIES FUEL IMAGINATION</h3>
+        </div>
+        <h3>SEE OUR LIBRARY RESOURCE</h3>
+      </section>
+      <!-- Featured Partcipants -->
+      <section class="investors font">
```

Comment:
- [ ] [OPTIONAL] Good job so far your contributor section is loading dynamically :clap:  But the design is a bit different compared to the original design. :call_me_hand:   Please make sure that your contributor section has use the same font, `height`, `width `, and spacing `padding/margin `. Please change it  to the color defined on the original design  in order to make your  design similar to the original design :+1: 
 Check the screenshots below :point_down:  to see the differences.

#### Orignal design
![Screenshot from 2022-10-28 18-19-46](https://user-images.githubusercontent.com/6615394/198707921-f81e8d82-3ba1-46d2-868b-a34d04408983.jpg)